### PR TITLE
sc-27337-jit-github-action-new-version-that-supports

### PR DESCRIPTION
**Pull request by avichayJit**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -270,20 +251,16 @@ runs:
       run: |
         echo CONFIG_FILE_PATH=${CONFIG_FILE_PATH} > /tmp/docker_env.txt
         echo SOURCE_CODE_FOLDER=/code >> /tmp/docker_env.txt
-        echo FAIL_ON_FINDINGS=${{ inputs.fail_on_findings }} >> /tmp/docker_env.txt
-        if [ -n "${{ inputs.security_control_output_file }}" ]
-        then
-          echo SECURITY_CONTROL_OUTPUT_FILE=${{ inputs.security_control_output_file }} >> /tmp/docker_env.txt
-        fi
+        echo FAIL_ON_FINDINGS=true >> /tmp/docker_env.txt
+        # TODO: Avichay - check if it ok to get rid of security_control_output_file
```
Comment (yardensade) [inline action.yml:255]: just reminding it's a public repo :D

Comment (adiattlan) [reply action.yml:255]: forgot to delete this :)

File: .github/workflows /deployment.yml
[the remark this answers is not in the document]
Comment (adiattlan) [reply .github/workflows /deployment.yml:1]: let's talk

[the remark this answers is not in the document]
Comment (avichayJit) [reply .github/workflows /deployment.yml:1]: let's talk

File: .github/workflows /deployment.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        uses: actions/create-release@v1
+        with:
+          tag_name: ${{ github.event.inputs.version }}
+          release_name: Release ${{ github.event.inputs.version }}
+          body: |
+            Version ${{ github.event.inputs.version }}
+          draft: false
+          prerelease: false
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (meshiyo) [inline .github/workflows /deployment.yml]: make sure that only Jit developer can release a new version

**Review by meshiyo — APPROVED** [review]

**Pull request merged**